compiler/ballerina-lang/src/test/java/io/ballerina/projects/test/resolution/packages/WorkspaceRepoDependencyTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.projects.test.resolution.packages;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * This class contains test cases that uses direct dependencies coming from local repos.
+ *
+ * @since 2.0.0
```

Comment:
```suggestion
 * @since 2201.13.0
```

Reply:
Fixed in [b472f53](https://github.com/ballerina-platform/ballerina-lang/pull/44229/commits/b472f5361e8761c1dc294d0ae9b472d4d4b884fe)